### DISCUSSION TITLE
How can I configure regexManager rules so that their PRs are not grouped with othes

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#21392] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I'd like the ability to specify if a regexManager rule should be raised in a separate PR, or grouped / combined with other updates.

We have a few rules that are updating Bitbucket Pipeline image statements from unauthenticated to authenticated image objects. 

Ideally, I'd like these to be considered as `updateType: replacement`.  However, at present they come through `pinDigest`, `digest`, `patch`, `minor` or `major` update types.

### What you tried so far.

I tried added a `groupName` but the renovate config validator reminded me that this is not supported.  I'm not aware of any other options available. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I'd suggest either changing depName, or setting additionalBranchPrefix

**Reply by setchy** [reply]
Looks like `additionalBranchPrefix` isn't supported within regexManager

```
"errors": [
    {
        "topic": "Configuration Error",
        "message": "Regex Manager contains disallowed fields: additionalBranchPrefix, matchUpdateTypes"
    }
]
```

**Reply by rarkins** [reply, marked as answer]
Can you use matchManagers=regex, additionalBranchPrefix="regex-" ?

**Reply by setchy** [reply]
@rarkins - this appears to have done the trick.  Will continue to monitor.  Thanks again
